### ISSUE TITLE
Colors: add contrast accessibility check for variants

### ISSUE THREAD

**Issue by fmaclen** [fmaclen/chromabox#27]
- Each variant should display: `7.84 AAA` (REF https://colorable.jxnblk.com/fffdfd/ff0000)
- The text should either be 100% white (or black) if the variant is too dark/light respectively
- I think this should be supported by [Colord a11y plugin](https://github.com/omgovich/colord?tab=readme-ov-file#plugins)
